client/my-sites/no-results/index.jsx

```diff
@@ -3,27 +3,25 @@
  */
 import React from 'react';
 
+/**
+ * Internal dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+
 /**
  * Style dependencies
  */
 import './style.scss';
 
-class NoResults extends React.Component {
-	static defaultProps = {
-		text: 'No results',
-		image: false,
-	};
-
-	render() {
-		return (
-			<div className="no-results">
-				{ this.props.image ? (
-					<img className="no-results__img" src={ this.props.image } alt="" />
-				) : null }
-				<span>{ this.props.text }</span>
-			</div>
-		);
+export default function NoResults( { text, image = false } ) {
+	const translate = useTranslate();
+	if ( text === undefined ) {
+		text = translate( 'No results' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **79** times:
`translate( 'No results.' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/no-results)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).